### DISCUSSION TITLE
How to Assign Specific Colors to Designated Nodes in a Tetrahedral Mesh Using SOFA19？

### DISCUSSION THREAD

**Discussion by sodasoda520** [sofa-framework/sofa#5028] [Write / Run a simulation]
Hello ,

I am working with SOFA19 to simulate a flexible object using a tetrahedral mesh generated from Gmsh. To better visualize the movement of specific nodes in the middle section of the object during the simulation, I want to assign a distinct green color to these designated nodes. However, when attempting to use addOrientedBoxRoi and setData to select and color these nodes, I encounter errors such as AttributeError: 'NoneType' object has no attribute 'value' and issues with creating SubsetMultiMapping. Could someone guide me on the correct method or alternative approaches in SOFA19 to assign specific colors to selected nodes in a tetrahedral mesh for enhanced visualization?

Thank you for your assistance!

**Comment by bakpaul** [comment]
Hello, 

I am sorry but I don't think you'll get a lot of help because of the SOFA version you are using. This is a 5 years-old release, we do not support it anymore. Try changing your SOFA version, it might solve your issue. 

Have a good day !

**Reply by sodasoda520** [reply]
Hello,

Thank you for your prompt response and for letting me know about the support status of SOFA19. I understand that it's an older release and appreciate the recommendation to upgrade. I will look into updating to a newer version of SOFA and see if that resolves the issues I'm encountering.

If I continue to face challenges after upgrading, I will reach out again for further assistance.

Thanks again for your help!
